warehouse/query-core/src/test/java/datawave/query/jexl/visitors/IvaratorRequiredVisitorTest.java

```diff
@@ -0,0 +1,40 @@
+package datawave.query.jexl.visitors;
+
+import datawave.query.jexl.JexlASTHelper;
+import org.apache.commons.jexl2.parser.ASTJexlScript;
+import org.apache.commons.jexl2.parser.ParseException;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class IvaratorRequiredVisitorTest {
+    
+    @Test
+    public void testExceededOrThresholdTrue() throws ParseException {
+        String query = "((_List_ = true) && (FOO == 1 && FOO == 3))";
+        ASTJexlScript script = JexlASTHelper.parseJexlQuery(query);
+        assertTrue(IvaratorRequiredVisitor.isIvaratorRequired(script));
+    }
+    
+    @Test
+    public void testExceededOrThresholdFalse() throws ParseException {
+        String query = "(FOO == 1 && FOO == 3)";
+        ASTJexlScript script = JexlASTHelper.parseJexlQuery(query);
+        assertFalse(IvaratorRequiredVisitor.isIvaratorRequired(script));
+    }
+    
+    @Test
+    public void testExceededValueThresholdTrue() throws ParseException {
+        String query = "((_Value_ = true) && (FOO == 1 && FOO == 3))";
+        ASTJexlScript script = JexlASTHelper.parseJexlQuery(query);
+        assertTrue(IvaratorRequiredVisitor.isIvaratorRequired(script));
+    }
+    
+    @Test
+    public void testExceededValueThresholdFalse() throws ParseException {
```

Comment:
Is this meant to be testing the same query as the one in line 21? If not, maybe adding some extra layer of complexity to the query to set it apart could be interesting.

Reply:
That's an extraneous oversight on my part - any query without the _Value_ or _List_ marker should return true so there isn't really a need for this test.